notebooks/002-datamodules/002-btech.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>This is a bit confusing. Masks are not needed for training, so why are they included in the train split?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>